MatrixTest.java

```diff
@@ -0,0 +1,163 @@
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotSame;
+import org.junit.Test;
+
+
+public class MatrixTest{
+    //Matrix(double[][] A){}
+
+    //Matrix(double[][] A, int m, int n){}
+    @Test
+    public void baseCtorSameLength(){
```

Comment:
Why are these tests called `baseCtor[other words]`? I don't have any idea what that means. It is a typo?